Bottom: 61017884f760eb63fda622e743c4d121dfa86244
Top:    ecef1363293505c494ad29af3819d0cd23511896
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-04 22:42:40 +0000

lesstif: Use Coord type for line-width, not int


---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index 795b9fe..c4b314e 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -52,7 +52,7 @@ typedef struct lesstif_gc_struct
 
   Pixel color;
   const char *colorname;
-  int width;
+  Coord width;
   EndCapStyle cap;
   char xor_set;
   char erase;
